### ISSUE TITLE
Mouse support

### ISSUE THREAD

**Issue by kidonng** [dundee/gdu#163]
Pardon me if this sounds like an unfair request for a TUI program, but it would be awesome to have mouse support like other modern TUI programs (think fzf, lazygit, etc.)

I noticed this project is using [tview](https://github.com/rivo/tview) under the hood, which seems to have mouse support baked-in, so it shouldn't be too hard to do.

**Comment by dundee** [comment]
Yes, you are right. It should be possible - https://github.com/rivo/tview/issues/13
